test/test_masked.py

```diff
@@ -162,6 +162,22 @@ def apply_masked_normalization_along_dim(op, input, *args, **kwargs):
 
 masked_ops = [op for op in op_db if op.name.startswith('_masked.')]
 masked_ops_with_references = [op for op in masked_ops if op.name.rsplit('.', 1)[-1] in reference_functions]
+masked_ops_with_mixed_layouts_support = [op for op in masked_ops if op.supports_sparse or op.supports_sparse_csr]
+
+
+def to_strided(obj):
+    """Convert the tensor content of object to strided tensor content.
+    """
+    if isinstance(obj, torch.Tensor):
+        if obj.layout == torch.strided:
+            return obj
+        return obj.to_dense()
+    elif isinstance(obj, (tuple, list)):
+        return type(obj)(list(map(to_strided, obj)))
```

Comment:
We have [pytree](https://github.com/pytorch/pytorch/blob/master/torch/utils/_pytree.py) in PyTorch that could simplify this (see an [example invocation](https://github.com/pytorch/pytorch/blob/c19cda5782f4c38140cb5c6aec08d5e44ec0c4cd/test/test_pytree.py#L131-L152))

Reply:
Thanks for the hint! Done.